package.json

```diff
@@ -28,12 +28,15 @@
     "@web3-react/portis-connector": "^6.0.2",
     "@zeit/next-css": "^1.0.1",
     "axios": "^0.19.0",
+    "babel-plugin-macros": "^2.8.0",
     "bootstrap": "^4.3.1",
     "ethers": "^4.0.41",
     "gh-pages": "^2.1.1",
     "graphql-request": "^1.8.2",
     "next": "^9.3.0",
+    "next-fonts": "^1.0.3",
     "next-images": "^1.3.1",
+    "prop-types": "^15.7.2",
     "react": "^16.13.0",
     "react-blockies": "^1.4.1",
     "react-bootstrap": "^1.0.0-beta.16",
```

Comment:
If we can, let's remove `web3-eth-contract` and `web3-utils`; they're probably not used, and we should prefer ethers over them anyway.